src/ExpressionEvaluator/Core/Test/ResultProvider/ResultProviderTestBase.cs

```diff
@@ -25,6 +25,20 @@ internal static string GetDynamicDebugViewEmptyMessage()
             return (string)emptyProperty.GetValue(exceptionType.Instantiate());
         }
 
+        internal static DkmClrCustomTypeInfo MakeCustomTypeInfo(params bool[] dynamicFlags)
+        {
+            if (dynamicFlags == null || dynamicFlags.Length == 0)
+            {
+                return null;
+            }
+
+            var builder = ArrayBuilder<bool>.GetInstance(dynamicFlags.Length);
+            builder.AddRange(dynamicFlags);
+            var result = CustomTypeInfo.Create(DynamicFlagsCustomTypeInfo.ToBytes(builder), tupleElementNames: null);
```

Comment:
In this case we want a `ReadOnlyCollection<byte>` rather than `ReadOnlyCollection<bool>` though. `ToBytes()` does the conversion.